.github/workflows/ci.yaml

```diff
@@ -20,10 +20,35 @@ env:
 
   GO_VERSION: "1.19"
   PYTHON_VERSION: "3.8.10"
-  NODE_VERSION: "16"
-  NPM_VERSION: "8.19.2"
+  NVM_VERSION: "0.39.3" # Does not need to be updated as often as the Node.js version or the NPM version
 
 jobs:
+
+  # ---- Configuration ----
+  config:
```

Comment:
We should name thos rather nvm-setup or similar because "config" is way to generic.

Reply:
My reasoning here was that this job can later be used for any "pre-staging" that we would need. So I chose a generic name. But maybe this is a case of YAGNI.